src/app.ts

```diff
@@ -19,7 +20,6 @@ import jwtPassport from './middleware/passport';
 const app = express();
 const PORT = process.env.PORT || 3000;
 
-app.use(passport.initialize());
 jwtPassport(passport);
```

Comment:
You don't use passport totally, you can remove it everywhere (package.json, middlewares)

Reply:
will do that, it was artefacts from my experiments)